src/Services/Spellcheckers/Aspell.php

```diff
@@ -64,7 +64,7 @@ public function check(string $text): array
      */
     private function getMisspellings(string $text): array
     {
-        $misspellings = iterator_to_array($this->run($text));
+        $misspellings = $this->run($text);
```

Comment:
Can we really drop this conversion at this position? 
Then it would be serialized as an iterator and returned as an iterator from the cache.

This would cause the `is_array()` call to always be `false`:

https://github.com/peckphp/peck/blob/818c85cefcf57eb2c1609ae584fdc249c10cfe51/src/Services/Spellcheckers/Aspell.php#L48-L53